src/SumoCoders/FrameworkCoreBundle/Menu/MenuBuilder.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace SumoCoders\FrameworkCoreBundle\Menu;
+
+use Knp\Menu\FactoryInterface;
+use Knp\Menu\ItemInterface;
+use SumoCoders\FrameworkCoreBundle\Event\ConfigureMenuEvent;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+class MenuBuilder
+{
+    /**
+     * @var EventDispatcherInterface
+     */
+    private $eventDispatcher;
+
+    /**
+     * @var FactoryInterface
+     */
+    private $factory;
+
+    /**
+     * @param \Symfony\Component\EventDispatcher\EventDispatcherInterface $eventDispatcher
```

Comment:
Can't you just use @param EventDispatcherInterface when the use statement is available?